### PR TITLE
tp-qemu: usb.cfg: add some usb types.

### PR DESCRIPTION
**Pull request by CongSmile**
Signed-off-by: Cong Li <coli@redhat.com>

### CODE REVIEW

**Comment by CongSmile** [comment]
id: 1079860

Need other PR: #2211 in virt-test.

File: qemu/tests/cfg/usb.cfg [hunk 1]
```diff
@@ -10,15 +10,39 @@
...
+                    usb_type_usbtest = piix3-usb-uhci
+                    usb_controller = uhci
+                    usb_max_port_usbtest = 2
+                    drive_format_stg = "usb1"
+                - piix4:
+                    usb_type_usbtest = piix4-usb-uhci
+                    usb_controller = uhci
+                    usb_max_port_usbtest = 2
+                    drive_format_stg = "usb1"
+                - uhci6:
```
Comment (xutian) [inline qemu/tests/cfg/usb.cfg]: why here is uhci6 not ich9-uhci6

Comment (CongSmile) [inline qemu/tests/cfg/usb.cfg]: Hi @xutian 

I have updated code, could you help review it again?

Thanks.

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by vipmike007** [comment]
Run following tests 
usb.usb_boot.usb_kbd.with_usb_hub.max_usb_dev.ich9-ehci usb.usb_boot.usb_kbd.with_usb_hub.max_usb_dev.usb-ehci
usb.usb_boot.usb_kbd.with_usb_hub.max_usb_dev.piix3-uhci
usb.usb_boot.usb_kbd.with_usb_hub.max_usb_dev.piix4-uhci
And Cli all looks good.

Based on above 
Acked-by: Mike Cao<bcao@redhat.com>

**Comment by xutian** [comment]
Merged. thanks @CongSmile  @suqinhuang  and @vipmike007

**Pull request merged**